NEPS/Config.cpp

```diff
@@ -1292,6 +1294,8 @@ static void to_json(json &j, const Config::Movement &o)
 	WRITE("Auto strafe", autoStrafe);
 	WRITE("Edge jump", edgeJump);
 	WRITE("Fast stop", fastStop);
+	WRITE("QuickPeekKey", QuickPeekKey);

```

Comment:
Here it is "QuickPeekKey" but when reading it is the same but with spaces "Quick Peek Key", this causes it to fail to load and reset to its default value